CONTRIBUTING.md

```diff
@@ -20,7 +20,11 @@ Ever since Moulberry has stopped working on NEU, other contributors have been wo
 
 ### Software configuration
 
-- Clone the NEU repository using `git clone https://github.com/NotEnoughUpdates/NotEnoughUpdates`.
+- Fork the NEU repository using the fork button on top right of the page and name the repo NotEnoughUpdates.
+- Clone the forked repository using `git clone https://github.com/<YourUserName>/NotEnoughUpdates`.
+- Make sure to create new branches for features you are working and not commit to the master branch of your repository.
+- After you have comitted all of the necessary changes make a pr on that branch.
```

Comment:
If we are writing it this thoroughly, might as well write out "pull request" rather than pr

Reply:
and add a comma after changes.

Reply:
Will do that